docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -78,49 +78,109 @@ Pick one of the following plugins:
 
 - [babel-plugin-import](https://github.com/ant-design/babel-plugin-import) with the following configuration:
 
+  `yarn add -D babel-plugin-import`
+
+  Create a `.babelrc` file in the root directory of your project:
```

Comment:
What about using the JavaScript format so we can keep the comment? Are the comments needed? What's the most popular format used by people, JSON or JavaScript to configure Babel?

Reply:
Great suggestion,  done.